package.json

```diff
@@ -96,6 +96,7 @@
     "@types/react": "^16.14.4",
     "@iov/crypto": "2.1.0",
     "axios": "^0.27.2",
+    "@keystonehq/bc-ur-registry": "^0.5.1",
```

Comment:
Is there any reason for using this library in global?

Reply:
This can delete.